.github/manimdependency.json

```diff
@@ -1,6 +1,6 @@
 {
     "windows": {
         "sox": "sox-14.4.2-win32",
-        "ffmpeg": "ffmpeg-4.3.1-win64-static"
+        "ffmpeg": "https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2020-09-24-12-32/ffmpeg-N-99357-g14d6838638-win64-gpl.zip"

```

Comment:
I don't particularly like that we are switching from `ffmpeg-4.3.1` to a nightly build. What about the released version that can be found at https://www.gyan.dev/ffmpeg/builds/ ?

Reply:
Just because it was Github. ;-)